README.md

```diff
@@ -1,16 +1,817 @@
+```
+
+## Conclusion
+
+Congratulations you've linked your conceptual grasp of Enumeration with the
+understanding of the code needed in Enumerables. Now would be a great time to
+go back to Ruby's [Enumerables documentation][enumdoc] and try to follow along
+with the code examples to see how you can use Enumerable methods to make your
+coding easier.
+
+[lm]: https://en.wikipedia.org/wiki/L%C3%A8se-majest%C3%A9
+
+#===============================================================================
+## The Enumerable Family Tree
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Use `map` to transform an `Array`
+* Use `reduce` to reduce an `Array` to a value
+* Use Ruby documentation to learn more about other variations on `map` and
+  `reduce`
 
 ## Introduction
 
-## SWBAT 1
+Now that you have written `map` and `reduce`, here's the big reveal: Ruby
+_already_ has these methods built into its `Array` data type!
+
+```ruby
+[10, 20, 30, 40].map{ |num| num * 2 } #=> [20, 40, 60, 80]
+[10, 20, 30, 40].reduce(0){ |total, num| total + num } #=> 100
+[10, 20, 30, 40].reduce(100){ |total, num| total + num } #=> 200
+```
+
+Now that you understand the "Character of Enumerable Methods" and the code
+behind using blocks, you are ready to unleash the full power of this module!
+Instead of merely repeating the documentation, we're going to help you apply
+your understanding to the `select` and `reject` methods, and then we're going
+to give you a list of document resources you should look up and master.
+
+These are the Enumerables you should ***memorize*** and practice heavily.
+They're going to be your friends every day in Ruby-land. There are other
+Enumerables that you might not memorize, but it's pretty common for a developer
+to realize that they're working in the "Character of Enumerable Methods" and
+think, "Hm, maybe there's an Enumerable for that...." When that moment hits,
+the developer comes _back_ to the documentation and look for that "just-right"
+Enumerable.
+
+## Select / Reject
+
+```ruby
+[10, 20, 30, 40].select{ |num| num > 25 } #=> [30, 40]
+[10, 20, 30, 40].reject{ |num| num > 25 } #=> [10, 20]
+```
+
+1. Map a collection
+2. Only accumulate the elements that make a truthy expression in the block for
+   `select`. Ruby also lets you do this work by calling the method `filter`,
+   which operates the same way.
+2. Only accumulate the elements that don't make a truthy expression in the
+   block for `reject`. You should see how they're basically the same method.
+
+## Enumerables to Memorize
+
+* `[all?][all?]`: Everything "tested" by the block returns truthy
+* `[any?][any?]`: Did anything "tested" by the block returns truthy
+* `[collect][collect]`: A synonym for `map`
+* `[count][count]`: Which elements satisfy the block _or_, without block, how many elements are there?
+* `[detect][detect]`: Which element satisfies the block _first_. Does the same thing as `find`.
+* `[find_all][find_all]`: Which elements satisfy the block?
```

Comment:
I didn't realize this was different than select but I guess it is: https://stackoverflow.com/questions/20999192/is-find-all-and-select-the-same-thing

Reply:
So no action to take here, right?